docs/proposals/ci-in-github-action.md

```diff
@@ -0,0 +1,76 @@
+# Proposal: CI in Github Action
+
+Last updated: 01.11.2021
+
+## Abstract
+
+Description of triggers and user actions to run CSI Baremetal CI.
+
+## Background
+
+We need to create Github Action to perform CSI Baremetal CI tests, 
+because running tests locally requires much hardware resources and developers' time.
+It is also making simpler to validate Pull Requests from fork-repositories.
+
+## Proposal
+
+We can make our tests running on Github Actions.
+
+Environment:
+- Ubuntu 20
+- Kind 0.11.1
```

Comment:
'--privileged' is needed in order to run e2e CI ?

Reply:
No, it works well even for LVG volumes. But I don't know how...